presto-product-tests/bin/run_on_docker.sh

```diff
@@ -159,6 +168,10 @@ docker version
 
 stop_all_containers
 
+if [[ "$CONTINUOUS_INTEGRATION" == 'true' ]]; then
```

Comment:
This won't work:
https://travis-ci.org/ArturGajowy/travis-docker-ec2-poc/builds/154320917#L265

Also, I think jenkins sets the same or similar variable for every build. Let me check :)